src/main/java/core/basesyntax/service/impl/TransactionParserImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionParserImpl implements TransactionParser {
+
+    private static final String DELIMITER = ",";
+    private List<String[]> strings;
+
+    @Override
+    public List<FruitTransaction> parseTransaction(List<String> string) {
+        List<FruitTransaction> listOfFruitTrsacion = new ArrayList<>();
+        strings = string.stream().map(words -> words.split(DELIMITER))
+                .collect(Collectors.toList());
+        for (String[] el : strings) {
+            FruitTransaction fruitTransaction = new FruitTransaction();
+            for (int i = 0; i < el.length; i += 3) {
+                fruitTransaction.setOperation((el[i]));
```

Comment:
hard to read what el[i] or el[i + 1] or el[i + 2] means?
use constants instead of numbers